common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -97,6 +97,15 @@ class ProctoringFields(object):
         scope=Scope.settings,
     )
 
+    hide_after_due = Boolean(
+        display_name=_("Hide Exam Results After Due Date"),
+        help=_(
+            "This setting overrides the default behavior of showing exam results after the due date has passed."
```

Comment:
@catong No, "Timed Exams" and "Proctored Exams" are treated separately, and the new feature only applies to the former. We refer to them with capitalized names in several places already (screenshots, also "Practice Exams" just below this line), making me think they're more proper nouns than general descriptions. Do you still want to remove the capitalization?

<img width="236" alt="screen shot 2016-04-28 at 10 20 06 am" src="https://cloud.githubusercontent.com/assets/7373924/14889065/2f1f8a16-0d2b-11e6-9f42-68f6b643bf03.png">
<img width="441" alt="screen shot 2016-04-28 at 10 19 51 am" src="https://cloud.githubusercontent.com/assets/7373924/14889064/2f1f088e-0d2b-11e6-85a3-d59d29a2d04d.png">